login/service.go

```diff
@@ -331,24 +331,6 @@ func (keycloak *KeycloakOAuthProvider) CreateOrUpdateIdentityAndUser(ctx context
 		return nil, nil, err
 	}
 
-	// new user for WIT
```

Comment:
ok, IIRC, so we want to avoid this block because if the user does not exist in `auth` when calling the `UpdateIdentityUsingUserInfoEndPoint` method below, then we don't want to create it in `wit`, correct?

Reply:
yeah, we would never create it on login. Only using the `Create User` API.

Reply:
ok